src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/index.js

```diff
@@ -105,13 +105,11 @@ class BoardingPassesInfo extends React.Component<Props> {
             <QueryRenderer
               query={nearestInfoBoardingPasses}
               render={this.renderBoardingPassesCard}
-              cacheConfig={{ force: true }}
```

Comment:
I had idea baggage info doesn't need to be refetched on every toggle 🤔

Reply:
What do you mean? I think if you delete this line, it will indeed be re-fetched every time, no?

Reply:
nope, it uses cache by default and `force: true` will enforce relay to call GraphQL

Reply:
Ah I see